include/votca/csg/pdbwriter.h

```diff
@@ -104,7 +108,7 @@ inline void PDBWriter::WriteContainer(T &container) {
   boost::format atomfrmt(
       "ATOM  %1$5d %2$-4s %3$-3s %4$1s%5$4d    %6$8.3f%7$8.3f%8$8.3f\n");
 
-  for (auto &atom : getIterable(container)) {
+  for (const auto &atom : getIterable(container)) {
```

Comment:
Jens this is the reason I pass in a unique_ptr to writeSymmetry() the beads are now stored as unique pointers in topology. When getIterable is called it is by default getting the begin and end bethods of the _beads container in topology. This means atom will be of type unique_ptr. I would have to store another container just with Bead * to do what you are asking unless I'm missing something?

Reply:
okay